src/settings/SettingsMenu.tsx

```diff
@@ -54,7 +54,7 @@ const SettingsMenu = ({ size, ...props }: SettingsMenuProps) => {
           isRound
         />
         <Portal>
-          <MenuList>
+          <MenuList zIndex={3}>
```

Comment:
Can we relate this to the other z-index? Perhaps add them alongside each other to zIndex.ts?

Reply:
I added the z-index for the code popups to zIndex.ts and used the existing `zIndexAboveTerminal` for the menus. This futureproofs the zIndex for the menus when we get round to collapsing the sidebar [`ea44b60`](https://github.com/microbit-foundation/python-editor-next/pull/609/commits/ea44b60e1b6a8301b30743ec88de9d99fb1c3352).

Reply:
Super, thanks.